video/out/gpu/video.c

```diff
@@ -313,6 +313,9 @@ static const struct gl_video_opts gl_video_opts_def = {
          .clamp = 1, }, // tscale
     },
     .scaler_resizes_only = true,
+    .correct_downscaling = true,
+    .linear_downscaling = true,
+    .sigmoid_upscaling = true,
```

Comment:
Does this need manpage change?

Reply:
It doesn't specify default value. Should I add it? It is not consistent where we specify default and where not.

Reply:
I would absolutely recommend adding it, for consistency. I think when the default is not written, it implies the flag is not enabled by default.